lib/LANraragi/Plugin/Metadata/Hdoujin.pm

```diff
@@ -1,107 +1,170 @@
-package LANraragi::Plugin::Metadata::Hdoujin;
+package LANraragi::Plugin::Metadata::HDoujin;
 
 use strict;
 use warnings;
 
-#Plugins can freely use all Perl packages already installed on the system
-#Try however to restrain yourself to the ones already installed for LRR (see tools/cpanfile) to avoid extra installations by the end-user.
+# Plugins can freely use all Perl packages already installed on the system
+# Try however to restrain yourself to the ones already installed for LRR (see tools/cpanfile) to avoid extra installations by the end-user.
 use Mojo::JSON qw(from_json);
 
-#You can also use the LRR Internal API when fitting.
+# You can also use the LRR Internal API when fitting.
 use LANraragi::Model::Plugins;
 use LANraragi::Utils::Logging qw(get_plugin_logger);
 use LANraragi::Utils::Archive qw(is_file_in_archive extract_file_from_archive);
 
-#Meta-information about your plugin.
+# Meta-information about your plugin.
 sub plugin_info {
 
     return (
-        #Standard metadata
-        name        => "Hdoujin",
+        # Standard metadata
+        name        => "HDoujin",
         type        => "metadata",
-        namespace   => "Hdoujinplugin",
-        author      => "Pao",
-        version     => "0.5",
-        description => "Collects metadata embedded into your archives by HDoujin Downloader's json or txt files.",
+        namespace   => "hdoujinplugin",
```

Comment:
I would avoid changing this namespace however since it's what's used for keeping track of user preferences